packages/fxa-auth-server/lib/payments/processing-tasks-setup.ts

```diff
@@ -76,10 +81,19 @@ export async function setupProcessingTaskObjects(processName: string) {
   const stripeHelper = new StripeHelper(log, config, statsd);
   Container.set(StripeHelper, stripeHelper);
 
+  const paypalClient = new PayPalClient(
```

Comment:
Is there any risk in adding the PayPal helper here? What happens if paypal config parameters aren't set? Can this function still be executed?

Reply:
there shouldnt be because i dont think we have any environment where stripe stuff is set but paypal isnt but ive moved this now into the remove-account script so if it is an issue it'd only be that script that breaks